libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp

```diff
@@ -35,89 +57,133 @@ AP_Volz_Protocol::AP_Volz_Protocol(void)
 
 void AP_Volz_Protocol::init(void)
 {
-    AP_SerialManager &serial_manager = AP::serialmanager();
-    port = serial_manager.find_serial(AP_SerialManager::SerialProtocol_Volz,0);
-    update_volz_bitmask(bitmask);
-}
-
-void AP_Volz_Protocol::update()
-{
-    if (!initialised) {
-        initialised = true;
-        init();
+    if (uint32_t(bitmask.get()) == 0) {
+        // No servos enabled
+        return;
     }
-    
+
+    const AP_SerialManager &serial_manager = AP::serialmanager();
+    port = serial_manager.find_serial(AP_SerialManager::SerialProtocol_Volz,0);
     if (port == nullptr) {
+        // No port configured
         return;
     }
 
-    if (last_used_bitmask != uint32_t(bitmask.get())) {
-        update_volz_bitmask(bitmask);
+    // Create thread to handle output
+    if (!hal.scheduler->thread_create(FUNCTOR_BIND_MEMBER(&AP_Volz_Protocol::loop, void),
+                                          "Volz",
+                                           1024, AP_HAL::Scheduler::PRIORITY_RCOUT, 1)) {
```

Comment:
i'd like to know how much stack space margin we have with this thread
check a log for the STAK messages, in MAVExplorer do "dump STAK"